Engine/RotoContext.cpp

```diff
@@ -4476,7 +4476,11 @@ RotoContextPrivate::bezulate(double time,
                 QPointF intersectionPoint;
                 for (; cur != polygon.end(); ++cur, ++last_pt) {
                     QLineF polygonSegment( QPointF(last_pt->x, last_pt->y), QPointF(cur->x, cur->y) );
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+                    if (line.intersects(polygonSegment, &intersectionPoint) == QLineF::BoundedIntersection) {
```

Comment:
I'm not a fan of having two open curly braces for one close brace in the code. Does clang-format handle this correctly?

Reply:
Instead, maybe add at the beginning of each of these 4 files:
```
#if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
// https://doc.qt.io/qt-5/qlinef-obsolete.html
#define intersects intersect
#define IntersectionType IntersectType 
#endif
```
I know this is kinda dangerous if "intersects" is used in the code but this seems ok to me

Reply:
`clang-format` can handle this indeed. As for the `define`s is better to use `using`s, although I prefer to not use them altogether.

Reply:
actually I don't think "using" is possible, since it's inside a class, not a namespace